Procfile

```diff
@@ -1 +1,2 @@
-web: gunicorn --worker-class uvicorn.workers.UvicornWorker -c run/gunicorn/devtest.py wsgi:app
+
+web: gunicorn --worker-class uvicorn.workers.UvicornWorker -c run/gunicorn/devtest.py --workers 3 wsgi:app
```

Comment:
@ksp37-dluhc  why do we need to increase the amount of workers ?

Reply:
Some of the routes in Assessment involve multiple (sometimes expensive) requests to the Assessment Store. So with a single worker, Assessment Store may only be able to process the requests sequentially. But with multiple workers it can process multiple requests at the same time (and so reduce the page load time in Assessment).